### DISCUSSION TITLE
Running rally locally

### DISCUSSION THREAD

**Discussion by kleenkanteen** [lukevella/rallly#776] [General]
I started up the landing page on port 3001:

![image](https://github.com/lukevella/rallly/assets/67878058/59dfb01f-34f4-4158-b092-bf6da3838c4d)

When I click "get started", it makes me log in:

![image](https://github.com/lukevella/rallly/assets/67878058/4da0aa66-dd3c-43b9-acc2-82333893de7d)

Why is this?

**Comment by lukevella** [comment]
Looks like `AUTH_REQUIRED` is `true` in your environment.

**Reply by kleenkanteen** [reply]
It is false: 

![image](https://github.com/lukevella/rallly/assets/67878058/d6ea610a-ae59-4261-9ef7-7e888091c909)

**Reply by kleenkanteen** [reply]
If you want to video call and I share my screen while you suggest changes, I can do so in an hour, 2:30 EST.

**Reply by kleenkanteen** [reply]
What does the confused emoji mean?

**Reply by lukevella** [reply]
Cmon man… I can't be spending time helping you set things up on your machine. I'm stretched thin as it is. I'm confused that you think that's a reasonable request.

**Comment by kleenkanteen** [comment]
I thought it would be a faster and more efficient use of your time in figuring out how to solve this issue instead of going back and forth here. I didnt know you are very busy...

**Reply by lukevella** [reply]
I understand. I unfortunately cannot spare any time and don't know any open source maintainers that could. It's best to assume this.

That said, I see what there was an issue that meant the middleware was not checking for `true` explicitly, so setting it to any value was enabling `AUTH_REQUIRED`. This is fixed here: https://github.com/lukevella/rallly/commit/177daa2aa1bb78ac46dd41265d263b5547b2725f
